frame/staking-rewards/src/benchmarking.rs

```diff
@@ -121,5 +133,27 @@ benchmarks! {
 
 	}: _(RawOrigin::Signed(user), position_id, validated_ratio)
 
+	reward_acumulation_hook_reward_update_calculation {
+		let now = T::UnixTime::now();
+
+		let reward = Reward {
+			asset_id: 1_u128.into(),
+			total_rewards: 0_u128.into(),
+			total_dilution_adjustment: 0.into(),
+			max_rewards: 100.into(),
+			reward_rate: RewardRate::per_second(6.into(), 10_000.try_into_validated().unwrap()),
+			last_updated_timestamp: now.as_secs(),
+			claimed_rewards: 0_u128.into()
+		};
+
+		let now = now + std::time::Duration::from_secs(12);
+	}: {
+		let reward = Pallet::<T>::reward_acumulation_hook_reward_update_calculation(1.into(), 1.into(), reward, now.as_secs());
```

Comment:
Can we benchmark `acumulate_rewards_hook` instead?

Reply:
Since the hook weight is completely dependant on the amount of pools and rewards, I figured this would give a more accurate weight